worker/src/lib.rs

```diff
@@ -42,15 +42,14 @@ static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 const SPRUCE_DIDWEB: &str = "did:web:tzprofiles.com";
 
 fn verify_signature(data: &str, pk: &JWK, sig: &str) -> Result<()> {
-    info!("Verify tweet signature.");
     let micheline_data = ssi::tzkey::encode_tezos_signed_message(data)?;
     let (algorithm, sig_bytes) = ssi::tzkey::decode_tzsig(sig)?;
     Ok(verify_bytes(algorithm, &micheline_data, pk, &sig_bytes)?)
 }
 
 fn initialize_logging() {
     use log::Level;
-    console_log::init_with_level(Level::Error).expect("error initializing log");
+    console_log::init_with_level(Level::Trace).expect("error initializing log");
```

Comment:
I think you should remove these changes as they're not relevant to the PR

Reply:
Done

Reply:
I meant to completely remove the changes from the PR, so this file has a clean Git history